src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -335,16 +335,16 @@ type ContainerProps = Omit<
 
 type ContainerState = {
   organizations?: Organization[];
-  selectedOrganization?: string;
+  selectedOrganization: string | null;
 };
 
 const ContextPickerModalContainer = createReactClass<ContainerProps, ContainerState>({
   displayName: 'ContextPickerModalContainer',
   mixins: [Reflux.connect(OrganizationsStore, 'organizations') as any],
   getInitialState() {
+    const storeState = OrganizationStore.get();
     return {
-      selectedOrganization:
-        OrganizationStore.organization && OrganizationStore.organization.slug,
+      selectedOrganization: storeState.organization && storeState.organization.slug,
```

Comment:
```suggestion
      selectedOrganization: storeState.organization?.slug,
```

Reply:
I applied this but as a separate commit as I also had to adjust the State type.